docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -356,14 +356,19 @@ Contributors are encouraged to run coverage on the test suite to identify areas
 that need additional tests. The coverage tool installation and use is described
 in :ref:`testing code coverage<topics-testing-code-coverage>`.
 
-Coverage should be run in a single process to obtain accurate statistics. To
-run coverage on the Django test suite using the standard test settings:
+To run coverage on the Django test suite using the standard test settings:
 
 .. console::
 
-   $ coverage run ./runtests.py --settings=test_sqlite --parallel=1
+   $ coverage run ./runtests.py --settings=test_sqlite
```

Comment:
Should we add `--concurrency=multiprocessing`? (see #15116) :thinking:

Reply:
I've added `concurrency = multiprocessing` in [`.coveragerc`](https://github.com/django/django/pull/16435/files#diff-4717170cf23369601199611189b7024c618ee859d198d90334e270c4f1cbdf36R3)

Reply:
Right, good :+1: